src/Microsoft.ML.PCA/PcaTrainer.cs

```diff
@@ -33,12 +33,31 @@ namespace Microsoft.ML.Trainers
     // REVIEW: make RFF transformer an option here.
```

Comment:
Do we need this REVIEW comment?

Reply:
Yes, this is not part of XML documentation, notice the double forward slash instead of triple?